updating/understanding-openshift-upgrade-duration.adoc

```diff
@@ -0,0 +1,96 @@
+[id="understanding-openshift-upgrade-duration"]
+= Understanding {product-title} upgrade duration
+include::_attributes/common-attributes.adoc[]
+:context: understanding-openshift-upgrade-duration
+
+toc::[]
+
+{product-title} upgrade duration varies based on the deployment topology. This section describes the approximate time duration of each component that follows through during the cluster update. It also explains the factors that affect the upgrade duration and the applicable {product-title} parameters.
+
+From {product-title} 4 onward, {op-system-first} is the only supported operating system (OS) for cluster upgrades. For a cluster with xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[{op-system-base} worker nodes] must be upgraded separately.
+
+== Prerequisites
+* You are familiar with xref:../architecture/architecture.adoc#architecture[OpenShift Container Platform architecture] and xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[OpenShift Container Platform updates].
+
+== Cluster upgrade duration
+
+Historical upgrade duration of similar clusters provides you the best estimate for the future cluster upgrades. However, if the historical data is not available, you should consider the time estimate of the following components to find your cluster upgrade duration:
+
+* Cluster Version Operator (CVO)
+* Machine Config Operator (MCO)
+* Pod Disruption Budget (PDB)
+* Cluster node health
+* Number of nodes in the cluster
+* {op-system-base-full} worker nodes
+
+=== Cluster Version Operator target upgrade payload deployment
+The Cluster Version Operator (CVO) retrieves the target upgrade release image and applies to the cluster. This process might take somewhere between 45 to 120 minutes.
+
+[NOTE]
+====
+The CVO process does not restart the workloads.
+====
+
+=== Machine Config Operator
+The Machine Config Operator (MCO) applies a new machine configuration to each control plane and worker node. During this process, the MCO performs the following actions on each node of the cluster:
+
+. Cordon and drain all nodes
+. Update the operating system (OS)
+. Node reboot
+. Uncordon all nodes and schedule workloads on the node
+
+[NOTE]
+====
+When a node is cordoned, it is unavailable to schedule a workload.
+====
+
+The control plane node upgrade duration is predictable and oftentimes shorter than worker nodes, because the control plane workloads are tuned for graceful updates and quick drains. You can upgrade the worker nodes in parallel by setting the `maxUnavailable` field to greater than 1 in the machine configuration pool. The MCO cordons the number of nodes specified in `maxUnavailable` and marks them unavailable for upgrade.
+
+The time to complete this process depends on many factors including the node and infrastructure configuration. This process might take 5 minutes to complete per node.
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
Not quite sure if this part is related or not with this section?

Reply:
@jiajliu Yes it is related to Machine Config additional resources.